bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -405,6 +405,9 @@ if len(files_for_combined_injfind) > 0:
     inj = wf.make_foundmissed_plot(workflow, found_inj, 
                 rdir['injections'], require='summ', tags=['ALLINJ'])
     inj_summ = list(layout.grouper(inj + sen, 2))
+
+# make analysis time summary
+layout.single_layout(rdir['analysis_time'], ([time_file,seg_summ_plot]))
                             
```

Comment:
Please use a single space after a comma, otherwise this is a fairly trivial patch. https://www.python.org/dev/peps/pep-0008/